modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -170,6 +176,7 @@ public void start() {
 
     @Override
     public void stop() throws Exception {
+        busyLock.block();
```

Comment:
Should we close versionTracker to unblock all the waiters?

Reply:
And is it ok to allow getting latest catalog version after CatalogManager was stopped?

We may violate invariant and the one will wait for a new catalog version, but the new version event will never be accepted due to stopping.

Reply:
done, tests fixed